definitions/npm/lodash_v4.x.x/flow_v0.63.x-/lodash_v4.x.x.js

```diff
@@ -3111,10 +3110,10 @@ declare module "lodash/fp" {
   declare export function defaultTo<T1: number, T2>(defaultValue: T2, value: T1): T1 | T2;
   declare export function defaultTo<T1: void | null, T2>(defaultValue: T2): (value: T1) => T2;
   declare export function defaultTo<T1: void | null, T2>(defaultValue: T2, value: T1): T2;
-  declare export var flow: HigherOrderFunction;
-  declare export var pipe: HigherOrderFunction;
-  declare export var flowRight: HigherOrderFunction;
-  declare export var compose: HigherOrderFunction;
+  declare export var flow: HigherOrderFunction | $ComposeReverse;
```

Comment:
I don't think this does what you want, this means that any invocation of this function must be compatible with both `HigherOrderFunction` and `$ComposeReverse`. Perhaps `&` is what you're looking for? in other words, I'm not sure how this doesn't fail tests/make this function less usable, but maybe I'm missing something?

Reply:
[Before](https://github.com/flowtype/flow-typed/pull/1793/files#diff-a986497c6415236202a140ff65ef7c06L3114) it was written as two functions with the same name as class methods.

I tried to recreate that functionality with `declare`s inside a module. But duplicated entries aren't possible. So I thought that's the equivalent. Are you sure that `HigherOrderFunction & $ComposeReverse` are the equivalent of above link?